packages/spark-core/base/inputs/_text-input.scss

```diff
@@ -0,0 +1,97 @@
+// ==================================================================
+// Input Component
+// ==================================================================
+.sprk-b-TextInput {
+  font-size: $font-size-body-two;
+  font-weight: $font-weight-body-two;
+  line-height: $line-height-body-two;
+  outline: 0;
+  padding: 3px $space-s;
+  box-shadow: $input-box-shadow;
+}
+
+.sprk-b-TextInput[disabled] {
+  background-color: $gray-light;
+  box-shadow: $input-disabled-box-shadow;
+  color: $input-gray-medium;
+}
+
+.sprk-b-TextInput--date-picker,
+.sprk-b-TextInput--monetary {
+  position: relative;
+}
+
+.sprk-b-DatePicker__icon,
+.sprk-b-TextInput--monetary::before {
+  position: absolute;
+  z-index: $layer-height-xs;
+}
+
+.sprk-b-DatePicker__icon {
+  margin-top: $input-icon-offset-y;
+  margin-left: $input-icon-offset-x;
+}
+
+.sprk-b-TextInput--monetary::before {
+  content: '$';
```

Comment:
I had an idea for an enhancement for theming on this. Could make the '$' a scss var. So, users who are using a different currency than ours or for whatever reason they can put whatever they want there.